flink-table/flink-table-uber/pom.xml

```diff
@@ -96,8 +95,8 @@ under the License.
 						<configuration>
 							<artifactSet>
 								<includes combine.children="append">
+									<!--Sql parser is included in planners-->
```

Comment:
Does it harm to include it here as well? Because you could also argue that `flink-table-api-java` is in `flink-table-api-scala` etc.

Reply:
It does. We do not relocate any packages in the `table-uber` jar. The sql-parser is relocated and included in the uber jar of  `table-planner*` modules. This is not the case for the api modules. Planners just declare dependency on those, but they do not bundle them into the jars